specs/deneb/p2p-interface.md

```diff
@@ -51,6 +50,8 @@ The specification of these changes continues in the same format as the network s
 | `MAX_REQUEST_BLOB_SIDECARS`              | `MAX_REQUEST_BLOCKS_DENEB * MAX_BLOBS_PER_BLOCK` | Maximum number of blob sidecars in a single request  |
 | `MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS`  | `2**12` (= 4096 epochs, ~18 days) | The minimum epoch range over which a node must serve blob sidecars  |
 | `BLOB_SIDECAR_SUBNET_COUNT`              | `6`                               | The number of blob sidecar subnets used in the gossipsub protocol.  |
+| `BLOB_KZG_COMMITMENTS_GINDEX`            | `2**4 + 11`  (= 27)               | `blob_kzg_commitments` field gindex on `BeaconBlockBody` container  |
```

Comment:
Is `BLOB_KZG_COMMITMENTS_GINDEX` unused?

Reply:
Please refer to the version at commit https://github.com/ethereum/consensus-specs/blob/8712451c43c60095eee2e10aaf996cf63710632a/specs/deneb/p2p-interface.md?plain=1#L54-L55 I would prefer to commit 8712451c43c60095eee2e10aaf996cf63710632a but the compilation does not prepend both lines in order, causing the used variable `BLOB_KZG_COMMITMENTS_GINDEX` to not be present

Reply:
I see. I'll push more commits to address it.

Reply:
I added some trick to bring `BLOB_KZG_COMMITMENTS_GINDEX` back and make it look smooth in the markdown file. (1657d16afd5d8def1b4f33999e7043f9ca7cd407)